### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by chandrashekar-cohesity**
Documentation First Draft.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,222 @@
...
+
+Lets go over the list of Workflows that this integration supports. 
+
+- [Helios Ransomware Alerts to SecureX and ServiceNow](#get-alerts-to-securex)
+- [Push Helios Ransomware Alerts to Private Intelligence](#push-alerts-to-pi)
+- [Ignore Anomaly on Cohesity Helios](#ignore-anomaly-workflow)
+- [Cohesity Restore Anomalous Object](#restore-anomaly)
+- [Push All Helios Ransomware Alerts to Private Intelligence](#push-all-anomaly)
+
+### <a name="get-alerts-to-securex"></a> Helios Ransomware Alerts to SecureX and ServiceNow
```
Comment (anvesh-cohesity) [inline README.md:110]: change the name to 'Helios Ransomware Alerts to Threat Response and ServiceNow'

File: README.md [hunk 2]
```diff
@@ -1 +1,222 @@
...
+| Threat Response API Client Password | Secure String | Threat Response API Client Password | Yes | 
+| Cohesity Helios API Key | Secure String | API Key to access Helios | Yes | 
+| Cohesity Helios Ransomware Alerts Filter  | String | Number of hours, used to get anomalous objects detected in the last `N` hours| Yes | 
+| Create ServiceNow Incidents | String | Set this variable to `yes` or `no` based on which ServiceNow incidents will be created | No | 
+
+##### Output
+
+N/A
+
+### <a name="push-alerts-to-pi"></a> Push Helios Ransomware Alerts to Private Intelligence
```
Comment (anvesh-cohesity) [inline README.md:131]: changed the name to 'Periodically Push Helios Ransomware Alerts to Threat Response'

File: README.md [hunk 3]
```diff
@@ -1 +1,222 @@
...
+- [Ignore Anomaly on Cohesity Helios](#ignore-anomaly-workflow)
+- [Cohesity Restore Anomalous Object](#restore-anomaly)
+- [Push All Helios Ransomware Alerts to Private Intelligence](#push-all-anomaly)
+
+### <a name="get-alerts-to-securex"></a> Helios Ransomware Alerts to SecureX and ServiceNow 
+[top](#Cisco-SecureX-Integration)
+
+This workflow pushes Cohesity Helios ransomware alerts to Threat Response Private Intelligence data store and also creates incidents on ServiceNow at regular intervals based on the schedule you define. 
+
+This workflow can also be triggered based on a schedule. The schedule that is used here is `Cohesity Helios Ransomware Data Push Schedule` which is a schedule to trigger orchestration workflow to push Helios ransomware data to private intelligence.
```
Comment (anvesh-cohesity) [inline README.md:115]: 'this workflow has to be triggered on a schedule' and user has to create `Cohesity Helios Ransomware Data Push Schedule` for this to work

File: README.md [hunk 4]
```diff
@@ -1 +1,222 @@
...
+
+N/A
+
+### <a name="push-alerts-to-pi"></a> Push Helios Ransomware Alerts to Private Intelligence
+[top](#Cisco-SecureX-Integration)
+
+This workflow pushes Cohesity Helios ransomware alerts detected in the last `N` hours to Threat Response private intelligence data store. 
+
+
+This workflow can also be triggered based on a schedule. The schedule that is used here is `Cohesity Helios Ransomware Data Push Schedule` which is a schedule to trigger orchestration workflow to push Helios ransomware data to private intelligence.
```
Comment (anvesh-cohesity) [inline README.md:137]: 'this workflow has to be triggered on a schedule' and user has to create `Cohesity Helios Ransomware Data Push Schedule` for this to work

File: README.md [hunk 5]
```diff
@@ -1 +1,222 @@
...
+
+N/A
+
+
+### <a name="ignore-anomaly-workflow"></a> Ignore Anomaly on Cohesity Helios
+[top](#Cisco-SecureX-Integration)
+
+This workflow ignores anomaly on for the specified object on Helios which removes this object from the list of anomalous objects. 
+
+> This is an Response workflows.
```
Comment (anvesh-cohesity) [inline README.md:158]: change 'workflows' to 'workflow'

File: README.md [hunk 6]
```diff
@@ -1 +1,222 @@
...
+
+##### Output
+
+N/A
+
+
+### <a name="ignore-anomaly-workflow"></a> Ignore Anomaly on Cohesity Helios
+[top](#Cisco-SecureX-Integration)
+
+This workflow ignores anomaly on for the specified object on Helios which removes this object from the list of anomalous objects.
```
Comment (anvesh-cohesity) [inline README.md:156]: 'ignores anomaly on for the' pls remove 'on';
This workflow ignores anomalous object alert on helios of the given object

File: README.md [hunk 7]
```diff
@@ -1 +1,222 @@
...
+| Cohesity Delete Sighting | String | Specifies where to delete or not the sighting once the anomaly is ignored. Can be `yes` or `no`. | Yes | 
+
+##### Output
+
+N/A
+
+### <a name="restore-anomaly"></a> Cohesity Restore Anomalous Object
+[top](#Cisco-SecureX-Integration)
+
+This workflow restores the specified anomalous object to the latest clean snapshot
```
Comment (anvesh-cohesity) [inline README.md:176]: mention that this workflow also resolves the alert on helios once the restore task is triggered. and the restored vm name will in the format Recover-{original VM name}-VM-{epoch timestamp when the vm is restored} and the restore task name on the cluster will be in format Cisco_SecureX_triggered_restore_task_{object name}

File: README.md [hunk 8]
```diff
@@ -1 +1,222 @@
...
+
+> This is an Response workflows.
```
Comment (anvesh-cohesity) [inline README.md:178]: replace 'workflows' with workflow

File: README.md [hunk 9]
```diff
@@ -1 +1,222 @@
...
+> This is an Response workflows. 
+
+##### Input 
+
+| **Argument Name** | **Type** | **Description** | **Required** |
+| --- | --- |--- | --- |
+| Threat Response API Client Id | Secure String | Threat Response API Client ID | Yes | 
+| Threat Response API Client Password | Secure String | Threat Response API Client Password | Yes | 
+| Cohesity Helios API Key | Secure String | API Key to access Helios | Yes | 
+| Delete | String | Specifies where to delete or not the sighting once the anomaly is ignored. Can be `yes` or `no`. | Yes |
```
Comment (anvesh-cohesity) [inline README.md:187]: argument name is 'Cohesity Delete Sighting'

File: README.md [hunk 10]
```diff
@@ -1 +1,222 @@
...
+- [Cohesity Restore Anomalous Object](#restore-anomaly)
+- [Push All Helios Ransomware Alerts to Private Intelligence](#push-all-anomaly)
+
+### <a name="get-alerts-to-securex"></a> Helios Ransomware Alerts to SecureX and ServiceNow 
+[top](#Cisco-SecureX-Integration)
+
+This workflow pushes Cohesity Helios ransomware alerts to Threat Response Private Intelligence data store and also creates incidents on ServiceNow at regular intervals based on the schedule you define. 
+
+This workflow can also be triggered based on a schedule. The schedule that is used here is `Cohesity Helios Ransomware Data Push Schedule` which is a schedule to trigger orchestration workflow to push Helios ransomware data to private intelligence.
+
```
Comment (anvesh-cohesity) [inline README.md:116]: this workflow also needs creation of 'ServiceNow_Credentials' under Account Keys and 'Cohesity_ServiceNow_Target' under Targets. These are needed for creating incidents on service now

File: README.md [hunk 11]
```diff
@@ -1 +1,222 @@
...
+| Threat Response API Client Id | Secure String | Threat Response API Client ID | Yes | 
+| Threat Response API Client Password | Secure String | Threat Response API Client Password | Yes | 
+| Cohesity Helios API Key | Secure String | API Key to access Helios | Yes | 
+| Delete | String | Specifies where to delete or not the sighting once the anomaly is ignored. Can be `yes` or `no`. | Yes | 
+
+##### Output
+
+N/A
+
+### <a name="push-all-anomaly"></a> Push All Helios Ransomware Alerts to Private Intelligence
```
Comment (anvesh-cohesity) [inline README.md:193]: changed the name to 'Push All Helios Ransomware Alerts to Threat Response'

**Review by anvesh-cohesity — COMMENTED** [review]
please make these changes before merging

File: README.md [hunk 12]
```diff
@@ -1 +1,222 @@
-# cisco-securex-orchestration-workflows
\ No newline at end of file
+# Cisco SecureX Integration
+
+<!--
+  Title: Cisco SecureX Integration
+  Description: This project is to integrate Cohesity Helios with Cisco SecureX and manage Cohesity Ransomware alerts on SecureX and take appropriate actions using SecureX dashboard.
```
[the remark this answers is not in the document]
Comment (chandrashekar-cohesity) [reply README.md:5]: Ok

File: README.md [hunk 13]
```diff
@@ -1 +1,222 @@
...
+  -->
+
+![](docs/assets/images/cohesity_ansible.png)
+
+## Overview
+
+This project is to integrate Cohesity Helios with Cisco SecureX and manage Cohesity Ransomware alerts on SecureX and take appropriate actions using SecureX dashboard.
```
[the remark this answers is not in the document]
Comment (chandrashekar-cohesity) [reply README.md:12]: done

File: README.md [hunk 14]
```diff
@@ -1 +1,222 @@
-# cisco-securex-orchestration-workflows
\ No newline at end of file
+# Cisco SecureX Integration
```
Comment (pyashish) [inline README.md:1]: Can the repo name be just SecureX , similar to : https://github.com/Tufin/SecureX

Comment (pyashish) [reply README.md:1]: @anvesh-cohesity  , we do the folder have subfolders ? should just be atomics/*.json and workflows/*.json ?

**Review by pyashish — APPROVED** [review]

File: README.md (hunk 14 above)
Comment (anvesh-cohesity) [reply README.md:1]: SecureX is broad name, it holds a lot of applications here. here we are mostly integrating with orchestration piece in SecureX, in future if we have to build an integration using relay server and have secureX tile dashboards.. or with amp endpoints , all these cannot go into a single repo SecureX. trying to keep them separate. 

and yes we have Atomics and Workflows folders with with json files in it. similar to Cisco's repo https://github.com/CiscoSecurity/sxo-05-security-workflows

**Pull request merged**